cfgmgr/Makefile.am

```diff
@@ -5,7 +5,7 @@ LIBNL_LIBS = -lnl-genl-3 -lnl-route-3 -lnl-3
 SAIMETA_LIBS = -lsaimeta -lsaimetadata -lzmq
 COMMON_LIBS = -lswsscommon -lpthread
 
-bin_PROGRAMS = vlanmgrd teammgrd portmgrd intfmgrd buffermgrd vrfmgrd nbrmgrd vxlanmgrd sflowmgrd natmgrd coppmgrd tunnelmgrd macsecmgrd fabricmgrd
+bin_PROGRAMS = vlanmgrd teammgrd portmgrd intfmgrd buffermgrd vrfmgrd nbrmgrd vxlanmgrd sflowmgrd natmgrd coppmgrd tunnelmgrd macsecmgrd fabricmgrd macvlanmgrd
```

Comment:
I would prefer vrrpmgrd, looks like we used macvlanmgrd in the HLD, please make the changes if possible.

Reply:
https://github.com/sonic-net/SONiC/pull/1446#issuecomment-2131108752

As suggested by @nser77, renaming vrrpmgrd to macvlanmgrd would enhance its generality within the whole SONiC architecture, making it more convenient for future functional components to use the macvlan interface type.

Reply:
What future functional components you are referring to here? the whole code is written for VRRP, wondering how you would use this macvlanmgr for other features.

Reply:
I'm not convinced with your answer, I don't see any mention of future functional components you are referring to here. You are trying to make the code changes with the assumption that some feature will make use of this macvlanmgrd.
If you really want to have a generic macvlanmgrd, remove VRRP name usage in the files and use it as a generic library so that any future module can use it, today VRRP feature can make use of generic MACVLAN mgr.

Reply:
Yes，you are right. We will use vrrpmgrd and have also submitted the PR for HLD https://github.com/sonic-net/SONiC/pull/1834. Could you please review it. Thanks.